modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/BasePython_conf.pm

```diff
@@ -0,0 +1,74 @@
+=pod
+=head1 NAME
+
+=head1 SYNOPSIS
+
+=head1 DESCRIPTION
+
+=head1 LICENSE
+    Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+    Copyright [2016-2021] EMBL-European Bioinformatics Institute
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+         http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and limitations under the License.
+=head1 CONTACT
+    Please subscribe to the Hive mailing list:  http://listserver.ebi.ac.uk/mailman/listinfo/ehive-users  to discuss Hive-related questions or to be notified of our updates
+=cut
+
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::BasePython_conf;
+
+use strict;
+use warnings;
+use Data::Dumper;
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use Bio::EnsEMBL::Hive::Version 2.5;
+
+use base ('Bio::EnsEMBL::Hive::PipeConfig::EnsemblGeneric_conf');
+
+
+sub default_options {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::default_options},
+
+        user  => $ENV{'USER'},
+        email => $ENV{'USER'}.'@ebi.ac.uk',
+
+        production_queue => 'production',
+        datamover_queue => 'datamover',
+    }
+}
+
+sub pipeline_create_commands {
+    my ($self) = @_;
+    return [
+        @{$self->SUPER::pipeline_create_commands},
+
+        # Create/alter tables used by ensembl.production.core.models.hive Python module
+        $self->db_cmd('CREATE TABLE result (job_id int(10), output TEXT, PRIMARY KEY (job_id))'),
```

Comment:
What's the reason for this? I have a contradicting ticket I created a while ago to remove those tables from standard Hive.py module in here ENSPROD-7139.

Reply:
As the comment at line 51 gives away, at the moment, if we remove those tables, several methods in `ensembl.production.core.models.hive` will break. This code requires table `result` to work, and the Celery task that will get jobs results will probably use parts of ensembl-prodinf-core ORM related to `result`.

Reply:
OK. We'll keep it until it's safe to delete then.